tests/AssetBundleTest.php

```diff
@@ -113,7 +113,11 @@ public function testBasePathEmptyWithAssetManagerWithBasePath(): void
 
     public function testBasePathEmptyBaseUrlEmptyWithAssetManagerWithBasePathWithBaseUrl(): void
     {
-        $manager = $this->manager->withLoader($this->loader->withBasePath('@asset')->withBaseUrl('@assetUrl'));
+        $manager = $this->manager->withLoader(
+            $this->loader
+            ->withBasePath('@asset')
```

Comment:
shouldn't it be bleeding?

```php
        $manager = $this->manager->withLoader(
            $this->loader
                ->withBasePath('@asset')
```

Reply:
Sry, I am new here :) . 
What do You mean with bleeding? Can you correct it here, so I can update and use this style in future corrections?

Reply:
Found. You are right. My mistake. Will make new PR.

Reply:
Thanks for the help 👍